tests/fakes.py

```diff
@@ -68,3 +81,47 @@
 echo -- "$@" >> "${log_path}/apt_get.args"
 env >> "${log_path}/apt_get.env"
 """
+
+LIVEPATCH_ENABLED_STATUS = """
+cat <<EOF
+client-version: "7.23"
+status:
+- kernel: 4.4.0-87.110-generic
+  running: true
+  livepatch:
+    checkState: {}
+    patchState: {}
+"""
+
+LIVEPATCH_CHECKED_UNAPPLIED = LIVEPATCH_ENABLED_STATUS.format(
+    'checked', 'unapplied')
```

Comment:
Per comment above, this would be something like `LIVEPATCH_ENABLED_STATUS.format(check_state='checked, patch_state='unapplied)`

Same below